drivers/tty/serdev/pibridge_trace.h

```diff
@@ -0,0 +1,315 @@
+#undef TRACE_SYSTEM
+#define TRACE_SYSTEM pibridge
+
+#if !defined(_PIBRIDGE_TRACE_H) || defined(TRACE_HEADER_MULTI_READ)
+#define _PIBRIDGE_TRACE_H
+
+#include <linux/tracepoint.h>
+#include <linux/serdev.h>
+#include <linux/kfifo.h>
+#include "pibridge.h"
```

Comment:
Nit: Separate `#include <...>` statements and the `#include "..."` statement with a newline.